CMakeLists.txt

```diff
@@ -636,6 +636,10 @@ if(USE_VULKAN)
   endif()
 endif()
 
+if(BUILD_LITE_INTERPRETER)
+  string(APPEND CMAKE_CXX_FLAGS " -DBUILD_LITE_INTERPRETER")
```

Comment:
There exists an BUILD_LITE_INTERPRETER option in https://github.com/pytorch/pytorch/blob/8f78face63cb3a14d53325ec1ddcf5d5d110f796/CMakeLists.txt#L152

Reply:
Also I believe the macros is added here https://github.com/pytorch/pytorch/blob/2b31cddfe5c95e3523cf08c0b596a506c84642cd/caffe2/CMakeLists.txt#L1098

Reply:
I checked commenting - it is not translated to binaries build